### PR TITLE
chore: tone down codecov

### PR DESCRIPTION
**Pull request by onerandomusername**

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -10,20 +10,33 @@ coverage:
...
+        only_pulls: true
+    patch:
+      default:
+        # basic
+        target: auto
+        threshold: 5%
+        base: auto
+        # advanced
+        branches:
+          - master
```
Comment (bast0006) [inline .codecov.yml:37]: We don't have a master branch, do we?

File: .codecov.yml [hunk 2]
```diff
@@ -10,20 +10,33 @@ coverage:
...
+        threshold: 5%
         base: auto
         flags:
           - unit
        # advanced settings
         branches:
           - main
+        if_ci_failed: ignore #success, failure, error, ignore
+        informational: true
+        only_pulls: true
```
Comment (bast0006) [inline .codecov.yml:28]: Also what does only pulls do here? Should that be commented or am I just ignorant on how this config works.

File: .codecov.yml [hunk 3]
```diff
@@ -40,4 +53,4 @@ comment:
   require_changes: no
 
 github_checks:
-  annotations: true
+  annotations: false
```
Comment (bast0006) [inline .codecov.yml:56]: Praise.

**Review by bast0006 — CHANGES_REQUESTED** [review]

File: .codecov.yml (hunk 2 above)
Comment (onerandomusername) [reply .codecov.yml:28]: Its the available options, since the docs example has those comments, but also does not list them as possible options later.

Comment (onerandomusername) [reply .codecov.yml:28]: Sorry, only replied to the last part of your comment.

"only_pulls" makes codecov not run on commits that aren't part of a pull yet.

**Review by bast0006 — APPROVED** [review]

**Pull request merged**